api/sessions.py

```diff
@@ -0,0 +1,33 @@
+from sqlalchemy import MetaData, create_engine
+from flask import Flask
+from sqlalchemy.orm import sessionmaker
+
+if __package__ is None or __package__ == '':
+  from settings import DefaultSettings
```

Comment:
just out of curiosity, what would this `if` condition do?

Reply:
In your last commit you had changed "from session import Sessions" to "from .session import Sessions", to look for that file relative to the current file. But that needs python to detect this directory as a package. I guess it's because I'm running windows, but for some reason despite the __init__.py it wasn't doing so. I thought if I change back to how I had initially written that import statement then it won't work for you and you'll have to change it again everytime you pull my code. So I just did this so that it checks if Python is detecting the package, if it is then it imports the way it's importing on your system, and if it doesn't detect a apackage then it imports the way it's working on my system. I hope this isn't a problem?

Reply:
Yeah, I had tried to. The "from .sessions import" way is the most commonly used approach, I don't know why it wasn't working at my end, it gave the error that python can't fine the __ main __.py , which according to StackOverflow essentially means it's not detecting it as a package. I tried all approaches but at the end found this solution that actually worked and ensured we don't have to change the import statements every time...